packages/core/src/components/tabs/_tabs.scss

```
@@ -6,28 +6,50 @@ $tabs-border-size: 1px;
 
 @include exports('ray-tabs') {
   .#{$ray-class-prefix}tabs {
-    border-bottom: $tabs-border-size solid $ray-color-blue-50;
-    display: flex;
+    &__list {
+      border-bottom: $tabs-border-size solid $ray-color-gray-10;
+      display: flex;
+    }
 
     &__item {
-      @extend .#{$ray-class-prefix}text--body-large;
+      font-size: 1rem;
+      padding-top: 0.75rem;
+      padding-bottom: 0.75rem;
       display: block;
       cursor: pointer;
       color: $ray-color-blue-50;
       text-align: center;
-      min-width: 200px;
-      padding-top: 1rem;
-      padding-bottom: 1rem;
       margin-bottom: -$tabs-border-size !important;
+      min-width: 85px;
+      outline: 0;
+
+      @media (min-width: map-get($ray-layout-grid-breakpoints, tablet)) {
```

prefer to use our mixins they're much more readable 
```suggestion
      @include ray-breakpoint(tablet)
```